core-tests/src/test/java/org/mule/runtime/core/mule/model/ReflectionEntryPointResolverTestCase.java

```diff
@@ -72,9 +73,10 @@ public void testExplicitMethodMatchComplexObject() throws Exception {
   public void testMethodMatchWithArguments() throws Exception {
     ReflectionEntryPointResolver resolver = new ReflectionEntryPointResolver();
     MuleEventContext eventContext =
-        new DefaultMuleEventContext(flowConstruct, Event.builder(DefaultEventContext.create(flowConstruct, TEST_CONNECTOR))
-            .message(of(new Object[] {new Apple(), new Banana()}))
-            .build());
+        new DefaultMuleEventContext(flowConstruct,
+                                    Event.builder(DefaultEventContext.create(flowConstruct, fromSingleComponent(TEST_CONNECTOR)))
```

Comment:
don hate me, but could you create a constant `TEST_CONNECTOR_LOCATION` where `TEST_CONNECTOR` is defined?